dockerfiles/Dockerfile.package

```diff
@@ -3,10 +3,11 @@ ARG RESTY_IMAGE_TAG="bionic"
 ARG DOCKER_KONG_SUFFIX
 ARG KONG_SHA
 ARG DOCKER_REPOSITORY
+ARG DOCKER_FPM_IMAGE=linuxforhealth/fpm:0.0.1
 

```

Comment:
The kong/fpm:0.0.1 image isn't built for s390x and using that repo's Makefile results in an error.  We can, however, run `docker build .` on linuxONE and successfully build the fpm image for s390x.  Added the ability to specify an FPM image to use.